src/main/java/org/apache/mesos/hdfs/executor/NameNodeExecutor.java

```diff
@@ -90,22 +90,20 @@ public void killTask(ExecutorDriver driver, TaskID taskId) {
   public void frameworkMessage(ExecutorDriver driver, byte[] msg) {
     log.info("Executor received framework message of length: " + msg.length + " bytes");
     String messageStr = new String(msg);
-    if (messageStr.equals(Scheduler.ACTIVATE_MESSAGE)) {
-      // Initialize the journal node and name node
-      runCommand(driver, nameNodeTask, "bin/hdfs-mesos-namenode -i");
-      // Start the primary name node
-      startProcess(driver, nameNodeTask);
-      driver.sendStatusUpdate(TaskStatus.newBuilder()
-          .setTaskId(nameNodeTask.taskInfo.getTaskId())
-          .setState(TaskState.TASK_RUNNING)
-          .build());
-      // Start the zkfc node
-      startProcess(driver, zkfcNodeTask);
-      driver.sendStatusUpdate(TaskStatus.newBuilder()
-          .setTaskId(zkfcNodeTask.taskInfo.getTaskId())
-          .setState(TaskState.TASK_RUNNING)
-          .build());
-    }
+    // Initialize the journal node and name node
+    runCommand(driver, nameNodeTask, "bin/hdfs-mesos-namenode -" + messageStr);
```

Comment:
Maybe it would be better for the message string to be a longer substring, like "-b" or "bin/hdfs-mesos-namenode -b". This also assumes we won't do anything else with a frameworkMessage in the future.